recipe/meta.yaml

```diff
@@ -34,22 +34,24 @@ requirements:
     - libtool        # [unix]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [osx]
+    - libgfortran  # [linux]
   run:
     - zlib 1.2*
+    - libgfortran  # [not win]
 
 test:
   requires:
-    - python 2.7.*  # [win and py27]
-    - python 3.4.*  # [win and py34]
-    - python 3.5.*  # [win and py35]
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
I thought this was already the case:

```
    - gcc  # [osx]
```

I think GitHub messed up the lines for the comments, @jakirkham can you clarify which line you were referring to?